src/api/APIProxy.ts

```diff
@@ -72,10 +74,33 @@ class APIProxy {
     data.me ? data.me.orders : null
   );
 
-  getOrderDetails = this.watchQuery(
-    QUERIES.OrderDetails,
-    data => data.orderByToken
-  );
+  getOrderDetails = (
```

Comment:
If those queries have different responses, shouldnt those be separate queries to avoid confusion? 

Docstring when you should one query and not another would be helpfull.

Reply:
I thought about making just one method for returning order details and depending on the internal SDK state which specifies if a user is logged in or not logged in it would automatically return appropriately different results. Therefore SDK would get rid of the need for determination of available order data based on the login state by the end developer, who implements SDK methods. SDK would make it internally automatically. What do you think about it?

Reply:
Ok, that would be consistent with the rest of the queries in the file.